fudge-ratio/src/main/webapp/about/about.css

```diff
@@ -2,52 +2,203 @@
 @import url('https://fonts.googleapis.com/css2?family=Bad+Script&family=Open+Sans:wght@400;700&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&family=Poiret+One&display=swap');
 :root {
-  --background-color: rgb(224, 239, 241);
-  --font-color: rgb(30, 30, 30);
+  --background-color: rgb(229, 231, 225);
   --accent-color:rgba(45, 93, 122, 0.712);
+  --block-color:rgb(158, 158, 2);
+  --base: min(3.5vw, 1.7rem);
 }
-
 body { 
-  margin:0; 
+  margin:0;
   background: var(--background-color);
 }
-
+/* Style for the bar on top and text on top*/
 .banner {
-  overflow: hidden;
-  /* 161, 111, 32, 0.712*/
+  z-index: 3;
```

Comment:
because when the banner scrolls over everyones pictures, it haze to say on top, so it needs a higher z index than the pictures and the triangles for it to stay on top. The text in the banner has the highest z index so that that is always on top of the banner and everything else in the page too